pom.xml

```diff
@@ -20,8 +20,8 @@
     <jackson.version>2.2.3</jackson.version>
     <jenkins.version>2.7.1</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
-    <node.version>5.8.0</node.version>
-    <npm.version>3.7.3</npm.version>
+    <node.version>6.4.0</node.version>
+    <npm.version>3.10.3</npm.version>
```

Comment:
@tfennelly ?? they are pulled down as binaries by the front end plugin. The slave environment shouldn't matter right?